src/Platform/Microsoft.Testing.Extensions.VSTestBridge/ObjectModel/ContextAdapterBase.cs

```diff
@@ -40,7 +40,7 @@ protected ContextAdapterBase(ICommandLineOptions commandLineOptions, IRunSetting
     // NOTE: Implementation is borrowed from VSTest
     // MSTest relies on this method existing and access it through reflection: https://github.com/microsoft/testfx/blob/main/src/Adapter/MSTest.TestAdapter/TestMethodFilter.cs#L115
     public ITestCaseFilterExpression? GetTestCaseFilter(
-        IEnumerable<string>? supportedProperties,
```

Comment:
This is a risky change to make given we do this:

https://github.com/microsoft/testfx/blob/c06959644eaf8132067c80d38d948f6df6c21327/src/Adapter/MSTest.TestAdapter/TestMethodFilter.cs#L124